sdk/messaging/azwebpubsub/ci.yml

```
@@ -0,0 +1,30 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/messaging/azwebpubsub/
+    - eng/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/messaging/azwebpubsub/
+
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'messaging/azwebpubsub/'
```

@jhendrixMSFT looks like the trailing slash is causing the changelog issue here (can't find/parse the package info since the regex won't match with the trailing slash).

```suggestion
    ServiceDirectory: 'messaging/azwebpubsub'
```

I can see about rolling in another update to just fail fast here or trim the slash since this is a non-obvious issue to debug.

---

Sounds good, thanks for taking a look.